lib/flow/VaporField.cpp

```diff
@@ -130,7 +136,6 @@ VaporField::GetFirstStepVelocityIntersection( glm::vec3& minxyz, glm::vec3& maxx
     for( int i = 0; i < 3; i++ )
     {
         grid = _getAGrid( 0, VelocityNames[i] );
-        VAssert( grid );
```

Comment:
It looks like we'll get a seg. fault here if a variable doesn't exist at time step 0.

Reply:
Fixed in commit e58098c .